src/Datadog.Trace.ClrProfiler.Native/environment_variables.h

```diff
@@ -67,6 +67,20 @@ const WSTRING log_path = "DD_TRACE_LOG_PATH"_W;
 // https://github.com/dotnet/coreclr/issues/12468
 const WSTRING clr_disable_optimizations = "DD_CLR_DISABLE_OPTIMIZATIONS"_W;
 
+// Indicates whether the profiler is running in the context
+// of Azure App Services
+const WSTRING azure_app_services = "DD_AZURE_APP_SERVICES"_W;
+
+// Gets the home directory of agent dependencies in azure app services
+const WSTRING azure_agent_home = "DD_AZURE_AGENT_HOME"_W;
```

Comment:
Is this new variable referenced anywhere?

Reply:
Not anymore!